test/suite/test_sweep04.py

```diff
@@ -0,0 +1,225 @@
+    ratio_examined = 0.01          # The chance that transient tables are examined
+    transient_table_max = 10000    # defines the length of the run
+    numkv = 1                      # Number of k/v pairs. Shouldn't matter for this test.
+    nsessions = 100                # Number of sessions in our pool.
+
+    conn_config = 'file_manager=(close_handle_minimum=0,' + \
+                  'close_idle_time=3,close_scan_interval=1),' + \
+                  'statistics=(fast),operation_tracking=(enabled=false),'
+
+    create_params = 'key_format=i,value_format=i'
+
+    def core_uri(self, i):
+        return '%s-c.%d' % (self.uri, i)
+
+    def transient_uri(self, i):
+        return '%s-t.%d' % (self.uri, i)
+
+    def create_table(self, uri):
+        self.session.create(uri, self.create_params)
+        c = self.session.open_cursor(uri, None)
+        for k in range(self.numkv):
+            c[k+1] = 1
+        c.close()
+
+    def examine(self, session, uri_maker, start, count):
+        for i in range(0, count):
+            c = session.open_cursor(uri_maker(start + i))
+            self.assertEquals(c[1], 1)
+            c.close()
+
+    def test_big_run(self):
+        # populate
+        r = suite_random()
+        for i in range(0, self.core_tables):
+            self.create_table(self.core_uri(i))
+
+        created_transient = 0    # running total of transients created
+        available_transient = 0  # the next available transient number
+
+        # Create the initial batch of transient tables
+        while created_transient < self.transient_tables:
+            self.create_table(self.transient_uri(created_transient))
+            created_transient += 1
+
+        # Open all the session we'll use in advance
+        sessions = []
+        for i in range(self.nsessions):
+            sessions.append(self.conn.open_session())
+
+        # We keep the dhandle counts for each time we get stats
+        dhandle_counts = []
+
+        # The big loop: For half the run, we are stressing by accessing both core tables
+        # and transient tables, and creating/dropping transient tables. The second half
+        # of the run, we stop creating/dropping tables and only access core tables
+        # just to see if all outstanding dhandles are swept.
+        maxloop = self.transient_table_max * 2
+        lasttime = time.time()
+        for loopcount in range(0, maxloop):
+            stressing = (created_transient < self.transient_table_max)
+            if loopcount % 100 == 0:
+                self.pr('{}/{}  stressing={}'.format(loopcount, maxloop, stressing))
+
+                # Make sure at least 3 seconds elapses between each 100 times through
+                # the loop, to give the various sweeps time to operate.
+                thistime = time.time()
+                delta = thistime - lasttime
+                if delta < 3.0:
+                    time.sleep(3.0 - delta)
```

Comment:
Could this '3.0' and the '3' in the config above be defined at the top so if we ever want to adjust to 2 or 4 seconds, it would be possibly two lines next to each other.